index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Dyary's Portfolio</title>
+        <link rel="stylesheet" href="styles.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div class="container">
+            <header>
+                <ul>
+                    <li id="my-logo-nav-item"><a href="#logo">My Logo</a></li>
+                    <li class="left-nav-items">Portfolio</li>
+                    <li class="left-nav-items">about</li>
+                    <li class="left-nav-items">contact</li>
+                    <li id="hamburger-nav-item">
+                        <img src="images/hamburger-icon.png" alt="hamburger
+                            icon">
+                    </li>
+                </ul>
+            </header>
+
+            <section id="hero-section">
+                <div id="hero-area-content">
+                    <header>
+                        <h1>
+                            I’m Dyary<br>
+                            Glad to see you!
+                        </h1>
+                        <p>I’m a software developer! I can help you build a
+                            product
+                            ,
+                            feature or website Look through some of my work and
+                            experience!
+                            If you like what you see and have a project you need
+                            coded,
+                            don’t hestiate to contact me.
+                        </p>
+
```

Comment:
- [ ] [OPTIONAL] Your `header` tag can finish there in the `line 41` after finishing your `p` tag.